src/app/components/README.md

```diff
@@ -0,0 +1,7 @@
+This repo follows the container component pattern. See ../containers/README.md for an explanation of this.
```

Comment:
This doesn't link to the containers Readme.

Reply:
@sareh Can I add a relative link for use on github and locally? i.e. this was deliberate

Reply:
@benjaminhobbs Yes, you can add it using the same syntax as a normal url `[Containers Readme](../containers/README.md)`

Reply:
No longer relevant due to changes